Cannot compile from a root directory

**Describe the bug**
When compiling inside a root directory such as `/OTP` compilation fails, but compilation succeeds inside `/OTP/subdir`.

**To Reproduce**

```
mkdir -p /OTP
wget -nv "https://github.com/erlang/otp/archive/OTP-24.0.tar.gz"
tar -zxf "OTP-24.0.tar.gz" -C /OTP --strip-components=1
cd /OTP
./otp_build autoconf
./configure
make

[...]

=== Entering application ssl
make[3]: Entering directory '/OTP/lib/ssl/src'
SED x86_64-pc-linux-gnu /OTP
 GEN    /OTP/lib/ssl/src/deps/ssl.d
make[3]: *** No rule to make target '../../../bootstrap/lib/public_key/includ../../..-PUB-KEY.hrl', needed by '../ebin/ssl_crl_cache_api.beam'.  Stop.
make[3]: Leaving directory '/OTP/lib/ssl/src'
make[2]: Leaving directory '/OTP/lib/ssl'
make[2]: *** [/OTP/make/otp_subdir.mk:29: opt] Error 2
make[1]: Leaving directory '/OTP/lib'
make[1]: *** [/OTP/make/otp_subdir.mk:29: opt] Error 2
make: *** [Makefile:498: libs] Error 2
```

**Expected behavior**
Compilation should work regardless of working directory.

**Affected versions**
OTP-24.0

**Additional context**
Add any other context about the problem here. If you wish to attach Erlang code you can either write it directly in the post using [code tags](https://guides.github.com/features/mastering-markdown/), create a [gist](https://gist.github.com), or attach it as a zip file to this post.

--- a/lib/ssl/prebuild.skip
+++ b/lib/ssl/prebuild.skip
@@ -1,1 +1,0 @@
-src/deps/ssl.d

--- a/lib/ssl/src/Makefile
+++ b/lib/ssl/src/Makefile
@@ -172,12 +172,16 @@ ERL_COMPILE_FLAGS += -I$(ERL_TOP)/lib/kernel/src \
 # Targets
 # ----------------------------------------------------
 
+opt debug: $(TARGET_FILES) $(APP_TARGET) $(APPUP_TARGET) $(DEP_FILE)
+
+deps: $(DEP_FILE)
+
 $(TARGET_FILES): $(BEHAVIOUR_TARGET_FILES)
 
 $(DEP_FILE): $(ERL_FILES)
 	@echo SED $(TARGET) $(ERL_TOP_NATIVE)
 	$(gen_verbose)erlc -M $(ERL_FILES) \
-	| sed "s@[a-zA-Z]\?$(ERL_TOP_NATIVE)@../../..@g" \
+	| sed "s@[a-zA-Z]\?$(ERL_TOP_NATIVE)/\(bootstrap/\)\?lib/\([^/]\+/\)@../../\2@g" \
 	| sed "s/\.$(EMULATOR)/\.$$\(EMULATOR\)/" \
 	| sed 's@^dtls_@$$(EBIN)/dtls_@' \
 	| sed 's@^inet_@$$(EBIN)/inet_@' \
@@ -185,7 +189,6 @@ $(DEP_FILE): $(ERL_FILES)
 	| sed 's@^tls_@$$(EBIN)/tls_@' \
 	> $(DEP_FILE)
 
-debug opt: $(TARGET_FILES) $(APP_TARGET) $(APPUP_TARGET) $(DEP_FILE)
 
 clean:
 	rm -f $(TARGET_FILES) $(APP_TARGET) $(APPUP_TARGET) $(BEHAVIOUR_TARGET_FILES) 

